modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/SocialDetail.js

```diff
@@ -100,21 +101,27 @@ export default function SocialDetail({
 
 	return (
 		<div className="c-p-3 traffic-source-detail">
-			<div className="c-mb-3 c-mt-2">
-				<TimeSpanSelector
-					disabledNextTimeSpan={chartState.timeSpanOffset === 0}
-					disabledPreviousPeriodButton={
-						isPreviousPeriodButtonDisabled
-					}
-					onNextTimeSpanClick={nextTimeSpan}
-					onPreviousTimeSpanClick={previousTimeSpan}
-					onTimeSpanChange={handleTimeSpanChange}
-					timeSpanKey={chartState.timeSpanKey}
-					timeSpanOptions={timeSpanOptions}
-				/>
-			</div>
-
-			{title && <h5 className="c-mb-4">{title}</h5>}
+			{!hideTimeSpanSelector && (
```

Comment:
If you follow my recommendation above, this could just be `{showTimeSpanSelector && ...`